rpc.if

```diff
@@ -132,6 +132,43 @@ interface(`rpc_write_exports',`
 	allow $1 exports_t:file write_file_perms;
 ')
 
+########################################
+## <summary>
+##	Allow adding/removing exports files
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`rpc_add_rm_exports',`
```

Comment:
Is it necessary or useful to discern between files and directories managing?
Interfaces names tend to use read/write/rw/manage/etc. in their second part.

Reply:
Interface removed